ext_mongodb.php

```diff
@@ -806,8 +811,16 @@ public function __debugInfo() : array
 
 	private int $milliseconds;
 
-	<<__Native>>
-	public function __construct(mixed $milliseconds);
+	public function __construct(mixed $milliseconds = NULL)
+	{
+		if ($milliseconds === NULL) {
+			$this->milliseconds = floor( microtime( true ) * 1000 );
+		} elseif (is_object($milliseconds) && get_class($milliseconds) == 'DateTime') {
```

Comment:
We had reopened the ticket for that already: https://jira.mongodb.org/browse/HHVM-162